--- org.eclipse.emf.doc/build/antJavadoc.sh.old	2009-04-18 16:39:29.000000000 +0100
+++ org.eclipse.emf.doc/build/antJavadoc.sh	2009-04-18 16:44:38.000000000 +0100
@@ -94,13 +94,13 @@
 # The directory of the plugins in the order they were built
 excludes="resources.jar";
 includes="${pluginName}";
-pluginDirs=`find $eclipseDir/plugins -name @dot -printf '%T@ %p\n' | sort -n | egrep "$includes" | egrep -v "$excludes" | cut -f2 -d' ' | sed -e 's/\(\/.*\)\/.*/\1/'`
+pluginDirs=`find -L $eclipseDir/plugins -name @dot -printf '%T@ %p\n' | sort -n | egrep "$includes" | egrep -v "$excludes" | cut -f2 -d' ' | sed -e 's/\(\/.*\)\/.*/\1/'`
 
 ### TODO: missing emf/xsd plugins (?) in $eclipseDir - need to copy them over or reference source (?)
 ### so that all classes/packages (and thus @links) can be resolved
 
 # All the jars in the plugins directory
-classpath=`find $eclipseDir/plugins \( -name "*.jar" -type f \) -o \( -name '@dot' -type d \) | tr '\n' ':'`; 
+classpath=`find -L $eclipseDir/plugins $eclipseDir/SDK/plugins \( -name "*.jar" -type f \) -o \( -name '@dot' -type d \) | tr '\n' ':'`; 
 if [[ $debug -gt 1 ]]; then echo "Got classpath: "; echo $classpath | perl -pe "s#:#\n\t:#g"; fi
 
 # Calculates the packagesets and the calls to copyDocFiles (used in ${antScript}.template)
--- org.eclipse.xsd.doc/build/antJavadoc.sh.old	2009-04-18 16:39:29.000000000 +0100
+++ org.eclipse.xsd.doc/build/antJavadoc.sh	2009-04-18 16:51:42.000000000 +0100
@@ -72,13 +72,13 @@
 # The directory of the plugins in the order they were built
 excludes="resources.jar";
 includes="org.eclipse.xsd|org.eclipse.emf.mapping.xsd2ecore";
-pluginDirs=`find $eclipseDir/plugins -name @dot -printf '%T@ %p\n' | sort -n | egrep "$includes" | egrep -v "$excludes" | cut -f2 -d' ' | sed -e 's/\(\/.*\)\/.*/\1/'`
+pluginDirs=`find -L $eclipseDir/plugins -name @dot -printf '%T@ %p\n' | sort -n | egrep "$includes" | egrep -v "$excludes" | cut -f2 -d' ' | sed -e 's/\(\/.*\)\/.*/\1/'`
 
 ### TODO: missing emf/xsd plugins (?) in $eclipseDir - need to copy them over or reference source (?)
 ### so that all classes/packages (and thus @links) can be resolved
 
 # All the jars in the plugins directory
-classpath=`find $eclipseDir/plugins \( -name "*.jar" -type f \) -o \( -name '@dot' -type d \) | tr '\n' ':'`; 
+classpath=`find -L $eclipseDir/plugins $eclipseDir/SDK/plugins \( -name "*.jar" -type f \) -o \( -name '@dot' -type d \) | tr '\n' ':'`; 
 if [ $debug -gt 1 ]; then echo "Got classpath: "; echo $classpath | perl -pe "s#:#\n\t:#g"; fi
 
 # Calculates the packagesets and the calls to copyDocFiles (used in ${antScript}.template)
